scripts/Deployment.ts

```diff
@@ -37,7 +37,10 @@ const ESCROW_PORTION = parseEther("1");
 const MAX_BONUS = parseEther("5");
 
 // Maximum duration that a lock can have
-const MAX_LOCK_DURATION = 60 * 60 * 24 * 365 * 4;
+const MAX_LOCK_DURATION = 60 * 60 * 24 * 365 * 3;
```

Comment:
Set to 3 years here and we use this figure in the curve calculation. Note that if the end date ends up being earlier than 3 years from the time of launch, nobody will be able to get the full multiplier.

Reply:
I saw that the end date is 1/21/2026, 8:00:00 AM
Should we allow some extra room then?
So people can get the full multiplier if they want, on the first few weeks at least?

Reply:
We will likely switch back to a linear model.